tests/query_test.go

```diff
@@ -21,16 +22,57 @@ func TestQuerySuite(t *testing.T) {
 	suite.Run(t, &QuerySuite{BaseTestSuite{initQueries: schema}})
 }
 
-func (s *QuerySuite) TestQueryFindById() {
-	store := NewResultSetFixtureStore(s.db)
+func (s *QuerySuite) TestQuery() {
+	store := NewQueryFixtureStore(s.db)
+	doc := NewQueryFixture("bar")
+	s.Nil(store.Insert(doc))
+
+	query := NewQueryFixtureQuery()
+	query.Where(kallax.Eq(Schema.QueryFixture.ID, doc.ID))
+	s.NotPanics(func() {
+		s.Equal("bar", store.MustFindOne(query).Foo)
+	}, "TODO: https://github.com/src-d/go-kallax/issues/49")
+
+	notID := kallax.NewID()
+	queryErr := NewQueryFixtureQuery()
+	queryErr.Where(kallax.Eq(Schema.QueryFixture.ID, notID))
+	s.Panics(func() {
+		s.Equal("bar", store.MustFindOne(queryErr).Foo)
+	})
+}
+
+// TODO: https://github.com/src-d/go-kallax/issues/39
+// this testcase can be removed once the .FindByID
+// has been autogenerated
+func (s *QuerySuite) TestExistFindByIdMethod() {
+	query := NewQueryFixtureQuery()
+	queryValue := reflect.ValueOf(query)
+	method := reflect.Indirect(queryValue).MethodByName("FindById")
+	s.True(method.IsValid(), "TODO: https://github.com/src-d/go-kallax/issues/39")
+}
 
-	doc := NewResultSetFixture("bar")
+// TODO: https://github.com/src-d/go-kallax/issues/39
```

Comment:
No, you understood me correctly :P. 

Then, if the code does not compile, delete it, or change it to test the behavior you want to test.

If you want you can keep the TODO comment to remind you in a future create a test for this case.